deploy/charts/trust-manager/values.yaml

```diff
@@ -78,7 +78,10 @@ app:
         # -- Whether to create an approver-policy CertificateRequestPolicy allowing auto-approval of the trust-manager webhook certificate. If you have approver-policy installed, you almost certainly want to enable this.
         enabled: false
 
-        # -- Namespace in which cert-manager was installed. Only used if approverPolicy has been enabled.
+        # -- Name of cert-manager's ServiceAccount. Only used if app.webhook.tls.approverPolicy.enabled is true
+        certManagerServiceAccount: "cert-manager"
```

Comment:
Nit: Maybe put this value below the `certManagerNamespace` value? Seems more conceptually correct to me. 😉